src/multiplex/client.rs

```diff
@@ -393,19 +387,17 @@ where
 
     fn call(&mut self, req: Request) -> Self::Future {
         let (tx, rx) = tokio::sync::oneshot::channel();
-        #[cfg(feature = "tracing")]
         let span = tracing::Span::current();
-        #[cfg(not(feature = "tracing"))]
-        let span = ();
-        event!(span, Level::TRACE, "issuing request");
+        span.in_scope(|| tracing::event!(Level::TRACE, "issuing request"));
```

Comment:
could be
```suggestion
        tracing::trace!(parent: &span, "issuing request");
```
although, if the span is current (as it was just returned by `Span::current()`) we may not _need_ to explicitly set it as the parent; the contract of the `Subscriber::current_span` method is that it should return the span that will be the parent of any event/span without an explicit parent at the current point in time...